actionview/test/template/form_helper_test.rb

```diff
@@ -168,7 +168,8 @@ def url_for(object)
     @url_for_options = object
 
     if object.is_a?(Hash) && object[:use_route].blank? && object[:controller].blank?
-      object.merge!(controller: "main", action: "index")
+      object[:controller] = "main"
+      object[:action] = "index"
```

Comment:
@oniofchaos same here

Reply:
FYI if you have rubocop gem, you should be warned when writing this, based on following rubocop configuration https://github.com/rails/rails/blob/master/.rubocop.yml#L128

Reply:
The enforced style is `double_quotes` ([docs](https://rubocop.readthedocs.io/en/latest/cops_style/#stylestringliterals)) so there should be no warning.